lib/fanv2_accessory.js

```diff
@@ -76,23 +110,36 @@ class Fanv2Accessory extends BaseAccessory {
         const hbSwing = this.tuyaParamToHomeBridge(Characteristic.SwingMode, this.swingMap.value);
         this.normalAsync(Characteristic.SwingMode, hbSwing, this.isRefresh)
       }
+
+      if (this.lightService && statusMap.code === 'light') {
+        this.switchLed = statusMap;
+        const hbLight = this.tuyaParamToHomeBridge(Characteristic.On, this.switchLed.value);
+        this.normalAsync(Characteristic.On, hbLight, this.isRefesh, this.lightService)
```

Comment:
`this.normalAsync(Characteristic.On, hbLight, this.isRefesh, this.lightService)` , Causes Characteristic to be created after each MQTT callback ,should use **this.isRefresh**